src/test/java/org/gridsuite/modification/server/modifications/tabularmodifications/TabularBatteryModificationsTest.java

```diff
@@ -97,4 +103,49 @@ protected void testUpdateModificationMessage(ModificationInfos modificationInfos
         Map<String, String> updatedValues = mapper.readValue(modificationInfos.getMessageValues(), new TypeReference<>() { });
         Assertions.assertEquals(ModificationType.BATTERY_MODIFICATION.name(), updatedValues.get("tabularModificationType"));
     }
+
+    @Test
+    public void testSqlRequestsCountOnGetModification() throws Exception {
+        UUID tabularWith1ModificationUuid = createTabularBatteryModification(1);
+        reset();
+        ApiUtils.getModification(mockMvc, tabularWith1ModificationUuid); // Getting one tabular modification with one sub-modification
+        assertSelectCount(3);
+
+        UUID tabularWith3ModificationUuid = createTabularBatteryModification(3);
+        reset();
+        ApiUtils.getModification(mockMvc, tabularWith3ModificationUuid); // Getting one tabular modification with three sub-modifications
+        assertSelectCount(3);
+    }
+
+    @Test
+    public void testSqlRequestsCountOnGetGroupModifications() throws Exception {
+        createTabularBatteryModification(1);
+        createTabularBatteryModification(3);
+
+        reset();
+        ApiUtils.getGroupModifications(mockMvc, getGroupId()); // Getting two tabular modifications with respectively one and three sub-modifications
+        assertSelectCount(6);
+    }
+
+    private UUID createTabularBatteryModification(int qty) {
+        ModificationInfos tabularModification = TabularModificationInfos.builder()
+            .modificationType(ModificationType.BATTERY_MODIFICATION)
+            .modifications(createBatteryModificationList(qty))
+            .build();
+        return saveModification(tabularModification);
+    }
+
+    private List<ModificationInfos> createBatteryModificationList(int qty) {
+        List<ModificationInfos> modifications = new ArrayList<>();
+        for (int i = 0; i <= qty; i++) {
+            modifications.add(
+                BatteryModificationInfos.builder()
```

Comment:
It could be nice to add some "reactiveCapabilityCurvePoints" to this builder to cover the @EntityGraph usage 
Same for other concerned tests